spyder/plugins/outlineexplorer/widgets.py

```diff
@@ -310,11 +330,47 @@ def remove_editor(self, editor):
                 except RuntimeError:
                     # item has already been removed
                     pass
-        
+
+    def set_editor_ids_order(self, ordered_editor_ids):
+        """
+        Order the root files items in the Outline Explorer following the 

```

Comment:
If we're not going to follow PEP 257, we shouldn't advertise that we do in our CONTRIBUTING guide.

Reply:
We agreed not to enforce it. So it's up to the contributor to follow it or not.